pypeit/coadd2d.py

```diff
@@ -850,6 +851,8 @@ def get_brightest_obj(self, specobjs_list, spat_ids):
                         ivar[:, iobj] = spec.OPT_COUNTS_IVAR
                         wave[:, iobj] = spec.OPT_WAVE
                         mask[:, iobj] = spec.OPT_MASK
+                    if np.any(np.isnan(flux)):
```

Comment:
I think this should be removed, correct?

Reply:
removed